### PR TITLE
[tenable_io] Updated Default value of Interval Paramter and Enabled Plugin Data Stream

### PR DESCRIPTION
**Pull request by mohitjha-elastic**
Type of change
- Enhancement

## What does this PR do?
Updated default value of interval parameter to 1h for Asset and Vulnerability and Enabled the plugin data stream.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally
Clone integrations repo.
Install elastic package locally.
Start elastic stack using elastic-package.
Move to integrations/packages/tenable_io directory.
Run the following command to run tests.

`elastic-package test`
## Automated Test
Attaching the test file below - 
[test-file.log](https://github.com/elastic/integrations/files/12671138/test-file.log)

### CODE REVIEW

**Review by marc-gr — APPROVED** [review]

File: packages/tenable_io/data_stream/plugin/manifest.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ title: Collect Plugin logs from Tenable Vulnerability Management
 type: logs
 streams:
   - input: httpjson
```
Comment (bhapas) [inline packages/tenable_io/data_stream/plugin/manifest.yml:4]: Shouldn't this datastream have same `1h` interval?

File: packages/tenable_io/data_stream/plugin/manifest.yml [hunk 2]
```diff
@@ -2,7 +2,6 @@ title: Collect Plugin logs from Tenable Vulnerability Management
...
-    enabled: false
     template_path: httpjson.yml.hbs
     title: Plugin logs
     description: Collect plugin logs from Tenable Vulnerability Management.
```
Comment (bhapas) [inline packages/tenable_io/data_stream/plugin/manifest.yml:7]: Shouldn't this datastream have same 1h interval?

File: packages/tenable_io/data_stream/plugin/manifest.yml (hunk 1 above)
Comment (mohitjha-elastic) [reply packages/tenable_io/data_stream/plugin/manifest.yml:4]: Hey @bhapas, 
The customer suggested setting the pull interval of assets and vulnerability to hourly and the plugin to 24hrs default.

**Review by bhapas — APPROVED** [review]

**Pull request merged**